docroot/themes/custom/mass_theme/overrides/js/table.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file
+ * Support responsive tables in rte
+ * Test samples:  /info-details/qag-info-details-table-samples
+ */
+
+// Apply to all .ma__table.
+(function ($) {
+  'use strict';
+
+  document.querySelectorAll('.ma__table').forEach(function (table) {
+    var hasRowHeaders = false;
+
+    // Set up assistive technology friendly tables.
+    // Add row scope for accessibility.
+    $(table).find('tbody th').each(function () {
+      $(this).attr('scope', 'row');
+      hasRowHeaders = true;
+    });
+
+    // Mobile format for table
+    if ($(table).has('thead th').length > 0) {
+      // 1. Set up mobile headers
+      var headerLabels = [];
+      $(table).find('thead th').each(function (headerIndex) {
+        // var headerLabel = $(this).text();
+        headerLabels.push($(this).text());
+      });
+
+      // Loop each row.
+      $(table).find('tbody tr').each(function () {
+        var targetCells = $(this).find('td');
+
+        $(targetCells).each(function (cellIndex) {
+          var headerIndex = cellIndex;
+
+          if (hasRowHeaders) {
+            headerIndex = cellIndex + 1;
+          }
+        });
+      });
+    }
+    else {
+      // No row headers
+      // Remove left space for headers with css.
+      if ($(window).width() < 781) {
+        $(table).addClass('no-headers');
+      }
+
+      $(window).on('resize', function () {
+        if ($(window).width() < 781) {
+          $(table).addClass('no-headers');
+        }
+      });
+    }
+
+
+    // Responsive tables
+    if ($(table).closest('.js-responsive-table')) {
+      // Copy captions authors entered into Mayflower template format.
+      // table-responsive.twig L.12
+      var userInputCaption = $(table).find('caption:not(.ma__table__caption)');
+      // 2nd test prevents empty caption content container. Otherwise add extra space at the top of the table.
+      if ($(userInputCaption) && $(userInputCaption).text().length > 0) {
+        var captionText = $(userInputCaption).text();
+        var captionSnippet =
+          '<span class="ma__table__caption__content">' +
+          captionText +
+          '</span>';
+
+        $(table).find('.ma__table__caption').prepend(captionSnippet);
```

Comment:
since the new scroll info is outside of the caption, we will need to readjust the js here to include `<caption>`